docs/docusaurus/docs/core/trigger_actions_based_on_results/create_a_custom_action.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Create a custom Action
+description: Run custom logic based on Validation Results to integrate with 3rd-party tools and business workflows.
+---
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+import PrereqPythonInstalled from '../_core_components/prerequisites/_python_installation.md';
+import PrereqGxInstalled from '../_core_components/prerequisites/_gx_installation.md';
+
+Great Expectations provides [Actions for common workflows](/application_integration_support.md#integrations) such as sending emails and updating Data Docs. If these don't meet your needs, you can create a custom Action to integrate with different tools or apply custom business logic based on Validation Results. Example use cases for custom Actions include:
+- Opening tickets in an issue tracker when Validation runs fail.
+- Sending emails to different teams depending on which Expectations fail.
```

Comment:
This has some overlap with our current email action right? Should we perhaps pick another example?